bin/pycbc_live

```diff
@@ -1169,13 +1175,21 @@ with ctx:
         results = {}
         evnt.live_detectors = set()
 
+        # Create objects to track whether the psd has been recalculated
+        if args.psd_variation:
```

Comment:
I don't particularly like the multiple `if args.psd_variation` statements thrown around at different points in this code but I can't think of how to generate these objects at the right time without them.

These could be generated regardless of the status of psd_variation but that feels wrong too.

Reply:
I've removed the first 2 `if args.psd_variation:` statements and had the `psd_recalculated` objects always be created and set whenever the psd has been recalculated.